heuristic/classes/Route.py

```diff
@@ -56,3 +56,72 @@ def _plan_cost(self, problem: Problem) -> float:
             cost += Stacks.cost(customer, problem, before, after)
 
         return cost
+
+    def is_feasible(self, problem: Problem, customer: int) -> bool:
+        """
+        Checks if inserting a customer in this route is feasible.
+        """
+        delivery = Item(problem.demands[customer], DEPOT, customer)
```

Comment:
```d_volume```, a `float` of just the delivery volume. No need for an Item here. Same for pickup, `p_volume`